deploy/bootstrap.sh

```diff
@@ -42,11 +42,6 @@ server_config=/var/www/biglotteryfund/deploy/server.conf
 APP_ENV_PLACEHOLDER="APP_ENV"
 sed -i "s|$APP_ENV_PLACEHOLDER|$APP_ENV|g" $server_config
 
-# Fix phantomjs install for Ubuntu
```

Comment:
Noticed this was stilling being run in `bootstrap` despite the fact we disabled PDF generation a while a go and no longer even install phantomjs